c-sharp-apps-IsaacKaz/transportation-app/PassengersTrain.cs

```diff
@@ -0,0 +1,53 @@
+﻿using c_sharp_apps_IsaacKaz.transportation_app;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace c_sharp_apps_IsaacKaz.transportation_app
+{
+
+    public class PassengersTrain : PublicVehicle
+    {
+        private Crone crone;
+        private int cronesAmount;
+        private int currentPassengers;
+
+        public PassengersTrain(int line, int id, int maxSpeed, Crone crone, int cronesAmount)
+            : base(line, id, maxSpeed, crone.GetSeats() * cronesAmount)
```

Comment:
You should calculate seats by each crone separately. Also, you should consider the extras.